src/components/homepage-header.tsx

```diff
@@ -0,0 +1,29 @@
+import React from "react";
+
+interface HomepageHeaderProps {
+  userName: string;
+  userImage: string;
+}
+
+export const HomepageHeader: React.FC<HomepageHeaderProps> = ({
```

Comment:
Please don't use React.FC (Chat-GPT??)... Write a regular TSX-function instead.

Reply:
sven wrote this but i´ll change it!